src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Connect Holland Secure JWT package and distributed under the terms of the MIT License.
+ * Copyright (c) 2020 Connect Holland.
```

Comment:
```suggestion
 * Copyright (c) 2020-2021 Connect Holland.
```

Please update codefixer as well